File: Algo/hr_implement/2/can_we_run_class.py
"""
Print a flag if there are students available for given criteria.
"""

def angryProfessor(k, a):
	on_timers = 0
	for i in a:
		if i < 1:
			on_timers +=1
	if on_timers >= k:
		return 'NO'
	else:
		return 'YES'

if __name__ == "__main__":
	k = 2
	a = map(int,'0 -1 2 1'.split(' '))
	#a = map(int,'26 94 -95 34 67 -97 17 52 1 86'.split(' '))
	print angryProfessor(k,a)

File: Algo/hr_implement/2/can_we_run_class.js
function angryProfessor(k,a) {
    let on_timers = 0;

    a.forEach(element => {
        if(element < 1) {
            on_timers++;
        }
    });
    
    if(on_timers>=k){
        return 'No';
    } else{
        return 'Yes';
    }
}

let k =2,
    a = '0 -1 2 1'.split(' ').map(Number);

console.log(angryProfessor(k,a));

